test/integration/select_test.lua

```diff
@@ -798,8 +826,8 @@ add('test_select_with_collations', function(g)
 
         local conditions = ...
 
-        local objects, err = crud.select('customers', conditions)
-        return objects, err
+        local result, err = crud.select('customers', conditions)
+        return crud.unflatten_rows(result.rows, result.metadata), err
```

Comment:
nit: in `simple_operations` tests rows are unflattened outside of netbox call.

Reply:
(and I like such approach more)

Reply:
Fixed